app/controllers/internal/organizations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Internal::OrganizationsController < Internal::ApplicationController
+  layout "internal"
+
+  def index
+    @organizations = Organization.order("name DESC").page(params[:page]).per(50)
+
+    return if params[:search].blank?
+
+    @organizations = @organization.where(
```

Comment:
I think `@organization` should be plural here `@organizations.where(`

Reply:
Lol yeah, I am about to push a spec for this, I noticed it and was annoyed that I didn't write a spec for search